plan/physical_plan_builder.go

```diff
@@ -406,9 +385,6 @@ func (p *Join) handleLeftJoin(prop *requiredProperty, innerJoin bool) (*physical
 			allLeft = false
 		}
 	}
-	if !allLeft {
```

Comment:
Make best effort to push down sort. Because if there is a limit and sort, it will be converted to top-n. Running top-n as early as possible will save running time.